internal/configload/configload.go

```diff
@@ -39,9 +39,10 @@ var DBs = map[string]string{
 	"ferretdb-sqlite-replset":         "mongodb://127.0.0.1:27002/?replicaSet=rs0",
 	"ferretdb-sqlite-replset-secured": "mongodb://username:password@127.0.0.1:27004/?authSource=admin&replicaSet=rs0",
 
-	"ferretdb2": "mongodb://127.0.0.1:47001/",
+	"ferretdb2":         "mongodb://127.0.0.1:47001/",
+	"ferretdb2-secured": "mongodb://username:password@127.0.0.1:47002/",
 
-	// add ferretdb2 secured and ferretdb2-dev versions
+	// add ferretdb2-dev version
 	// TODO https://github.com/FerretDB/dance/issues/1154
```

Comment:
Would adding ferretdb2-dev and/or ferretdb2-dev-secured version be sufficient for this issue?

Reply:
I think we need both (in the next PR)